Autocomplete/predefined-items-and-recent-searches/src/predefined-items.js

```diff
@@ -0,0 +1,51 @@
+export const predefinedItems = {
```

Comment:
That's interesting to treat this as a plugin 🙂

Reply:
yeah, everything can be a plugin :)